src/renderer/containers/ChooseWorkspaceDialog.tsx

```diff
@@ -37,23 +42,36 @@ function mapStateToProps(state: State, ownProps: IChooseWorkspaceDialogOwnProps)
     }
     workspaceDir = isLocalWebAPI ? workspaceDir || '' : null;
     workspaceName = workspaceName || '';
+    let workspaceNames: string[];
+    if (state.data.workspace && state.data.workspaceNames) {
+        let workspaceBaseDir = state.data.workspace.baseDir;
```

Comment:
Later, we want the API to return just the name in `state.data.workspace.baseDir`, so we don't need the following lines of code, right?

Reply:
yes